ui/src/components/workflow/executions/WorkflowList.js

```diff
@@ -209,11 +225,85 @@ const Workflow = React.createClass({
     this.state.update = true;
     this.refreshResults();
   },
+
+  handleRowSelect(row, isSelected, e) {
+    let currWFEs = this.state.selectedWFEs;
+    this.state.update = true;
```

Comment:
I think best practice would want you to use setState instead of setting the property directly.

Reply:
Refactored in latest commit

Reply:
And prefer destructuring, try:

const { selectedWFEs = []  } = this.state;